### DISCUSSION TITLE
How to redirect FastAPI to localhost:8080/api instead of localhost:8080/docs

### DISCUSSION THREAD

**Discussion by anthonybe98** [fastapi/fastapi#6584] [Questions]
### First Check

- [x] I added a very descriptive title to this issue.
- [x] I used the GitHub search to find a similar issue and didn't find it.
- [x] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [ ] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import  FastAPI
from typing import Optional
import uvicorn
from pydantic import BaseModel
from fastapi.encoders import jsonable_encoder
from typing import List

app = FastAPI()

people = []
class Item(BaseModel):
    id: str
    firstname: str
    familyname: str
    phonenumber: str
    email: str


@app.get("api/people", response_model = List[Item])
def get_people():
    return people
if __name__ == "__main__":
    uvicorn.run(app, host="localhost", port=8080)
```

### Description

When I run the code I would ideally want my api to be reached at localhost:8080, then if I want to do a get I want to reach it at  localhost:8080/api/people. Is it doable?

### Operating System

macOS

### FastAPI Version

0.63.0

### Python Version

3.8

**Comment by mawassk** [comment]
You can specify `docs_url` when creating an instance of `FastAPI`. Just do 
```python
app = FastAPI(docs_url="/api")
```

**Comment by raphaelauv** [comment]
@anthonybe98 could you close the issue , thank you

**Comment by YuriiMotov** [comment]
To make initial code working you need to add a slash in the beginnings of the path:
```
@app.get("/api/people", response_model = List[Item])  #  `/api/people` instead of `api/people`
```
